File: Heat/ViewModels/Plants/DetailsServicePlantViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Plants

    Public Class DetailsServicePlantViewModel

        Property ID As Integer

        Property PlantID As Integer


        <Display(Name:="Data ultima manutenzione eseguita")>
        <DisplayFormat(DataFormatString:="{0:d}")>
        Property PreviousServiceDate As Nullable(Of DateTime)

        <Display(Name:="Periodicità")>
        Property Periodicity As Periodicity

        <Display(Name:="Scadenza")>
        <DisplayFormat(DataFormatString:="{0:d}")>
        Property LegalExpirationDate As DateTime

        <Display(Name:="Data programmata della manutenzione")>
        <DisplayFormat(DataFormatString:="{0:d}")>
        Property PlannedServiceDate As DateTime
    End Class
End Namespace

File: Heat.ConvertedToC#/ViewModels/Plants/DetailsServicePlantViewModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Plants
{

    public class DetailsServicePlantViewModel
	{

		public int ID { get; set; }

		public int PlantID { get; set; }


		[Display(Name = "Data ultima manutenzione eseguita")]
		[DisplayFormat(DataFormatString = "{0:d}")]
		public Nullable<DateTime> PreviousServiceDate { get; set; }

		[Display(Name = "Periodicità")]
		public Periodicity Periodicity { get; set; }

		[Display(Name = "Scadenza")]
		[DisplayFormat(DataFormatString = "{0:d}")]
		public DateTime LegalExpirationDate { get; set; }

		[Display(Name = "Data programmata della manutenzione")]
		[DisplayFormat(DataFormatString = "{0:d}")]
		public DateTime PlannedServiceDate { get; set; }
	}
}
